src/common/components/ModalsManager.js

```diff
@@ -6,6 +6,7 @@ import styled from "styled-components";
 // import LoginHelper from "components/modals/LoginHelper";
 import AddInstance from "../modals/AddInstance";
 import Settings from "../modals/Settings/Settings";
+import ShowScreenShot from "../modals/ShowScreenShot";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "../modals/ShowScreenShot"](https://app.codacy.com/manual/gorilla-devs/GDLauncher/pullRequest?prid=5082094)